manifests/record/ptr/by_ip.pp

```diff
@@ -76,36 +76,78 @@
 
   # split the IP address up into three host/zone pairs based on class A, B, or C splits:
   # For IP address A.B.C.D,
-  # class C => host D / zone C.B.A.IN-ADDR.ARPA
-  # class B => host D.C / zone B.A.IN-ADDR.ARPA
-  # class A => host D.C.B / zone A.IN-ADDR.ARPA
-  $class_c_zone = inline_template('<%= @ip.split(".").reverse()[1..3].join(".") %>.IN-ADDR.ARPA')
+  # class C => host D / zone C.B.A.in-addr.arpa
+  # class B => host D.C / zone B.A.in-addr.arpa
+  # class A => host D.C.B / zone A.in-addr.arpa
+  $class_c_zone = inline_template('<%= @ip.split(".").reverse()[1..3].join(".") %>.in-addr.arpa')
```

Comment:
You can rename zone name from uppercase|camelcase to lowercase and your zone will be corrected.
I think is not good way use different registers names for one zone.
If we will use names only lowercase, we can guarantee for one zone exists one file of zone. We can't declare same zones:
```
  dns::zone { 'example.com':
    soa         => 'ns1.example.com',
    soa_email   => 'admin.example.com',
    nameservers => ['ns1']
  }
  dns::zone { 'eXample.Com':
    soa         => 'ns1.example.com',
    soa_email   => 'admin.example.com',
    nameservers => ['ns1']
  }
```
For reverse twin zones:
```
dns::zone { "168.192.in-addr.arpa":
  reverse => false,
}
dns::zone { "192.168":
  reverse => "reverse",
}
```
we have puppet error: "Duplicate declaration: Exec[bump-168.192.in-addr.arpa-serial] is already declared in file..." 
https://github.com/ajjahn/puppet-dns/pull/179/files#diff-2e38170d1d92d7eafada3bbc9bf83100R385